draft-ietf-rtcweb-jsep.xml

```diff
@@ -1901,8 +1962,9 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
               <t>A direction attribute which is the same as that of the
               associated transceiver.</t>
 
-              <t>For each supported codec, "a=rtpmap" and "a=fmtp"
-              lines, as specified in
+              <t>For each supported codec,
+              not excluded by the codec preferences of associated transceiver,
```

Comment:
For each supported codec **that has not been** excluded by the codec preferences of **the** associated transceiver